browser-test/src/applicant/application_download.test.ts

```diff
@@ -242,14 +242,14 @@ test.describe('normal application flow', () => {
     await adminPrograms.viewApplications(programName)
     const postEditCsvContent = await adminPrograms.getCsv(noApplyFilters)
     expect(postEditCsvContent).toContain(
-      'sarah,,smith,op2_admin,05/10/2021,1000.00',
+      'sarah,,smith,,op2_admin,05/10/2021,1000.00',
```

Comment:
**question (non-blocking):** Is it possible to include a suffix in one of these test cases rather than having it be blank in both?

Reply:
We will def add some test cases to include suffix when I actually make some frontend changes. It's difficult now, for example methods like below we can't even apply a `.cf-name-suffix` since it's not even created yet. https://github.com/civiform/civiform/blob/fb32344e118fc977609de00fd1deb34e44e7082f/browser-test/src/support/applicant_questions.ts#L60